### ISSUE TITLE
Add improved undo and redo

### ISSUE THREAD

**Issue by tmpg** [ddvk/remarkable-hacks#133]
Want to say thank you for all the work you all have been doing on this repo.

I have an RM2 and I can not get the undo and redo to work consistently with swiping left and right respectively in the top corner.

I use Adobe Fresco a lot and they have a double 2-finger tap for undo and a double 3-finger tap for a redo. It works well and does not break workflow. 

Would you guys consider implementing this functionality?

**Comment by unphased** [comment]
It's along the top inch or so, not the corner. doesn't even really work for me in the corner. swipe left and right along the top works much better. 

multitouch tapping does sound good though.

**Comment by tmpg** [comment]
I tried on the top and it is still pretty inconsistant. It does work better than corner but if multi-touch is supported I would vote on an effort to include support for something like what I described if it works well on e-paper.

**Comment by unphased** [comment]
I agree it's not really reliable enough to use... it seems like once you get in the groove of it it does work somewhat reliably and also in quick succession. but it can be a real challenge getting it to start working. 

This doesn't compare to how difficult it is to make the pinch to zoom gesture work though.

**Comment by ddvk** [comment]
i ll see what i can do

**Comment by tmpg** [comment]
I would love the feature but definitely should be a low priority. Thank you again for your work! I love the feature set!
